tools/js-tools/eslintrc/playwright.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @type {import("eslint").Linter.Config}
+ */
```

Comment:
Is this junk really needed here? This file shouldn't be being linted or otherwise processed as typescript in the first place.

Same for all the other files where you added this.

Reply:
It's not "junk", rather it's simple JSDoc comment and it has nothing to do with TypeScript. It simply helps with auto-completion of the config.

#### This is with that JSDoc comment

<img width="958" alt="Screenshot 2022-06-01 at 10 34 03 AM" src="https://user-images.githubusercontent.com/18226415/171331676-f06cf4a8-ac23-4ee9-a2c7-d97d820354c2.png">


#### This is without that JSDoc comment

<img width="976" alt="Screenshot 2022-06-01 at 10 34 45 AM" src="https://user-images.githubusercontent.com/18226415/171331697-2575c395-c07b-40a1-90b0-d92a3ffdd4b4.png">

Reply:
JSDoc is not a VS Code thing, it's used by every IDE that supports JavaScript. I would always love my IDE to provide that "little completion" help to make my job easier.

Anyways, I have cleaned up all the "junk" here. Thank you.

Reply:
I went ahead and raised the question for the team in Slack (you probably saw the ping). Let's get some other opinions, and if it's decided this would be useful we can do it in a separate PR.